src/app/accounts/change-password.component.ts

```diff
@@ -19,19 +19,25 @@ import { UserService } from 'jslib/abstractions/user.service';
 import { CipherString } from 'jslib/models/domain/cipherString';
 import { SymmetricCryptoKey } from 'jslib/models/domain/symmetricCryptoKey';
 
+import { KeysRequest } from 'jslib/models/request/keysRequest';
 import { SetPasswordRequest } from 'jslib/models/request/setPasswordRequest';
 
 import {
     ChangePasswordComponent as BaseChangePasswordComponent,
 } from 'jslib/angular/components/change-password.component';
 
+import { KdfType } from 'jslib/enums/kdfType';
+
 @Component({
     selector: 'app-accounts-change-password',
     templateUrl: 'change-password.component.html',
 })
 export class ChangePasswordComponent extends BaseChangePasswordComponent {
+    showPassword: boolean = false;
+    hint: string = '';
+
     onSuccessfulChangePassword: () => Promise<any>;
-    successRoute = 'lock';
+    successRoute = 'vault';
```

Comment:
Can you explain why this is changing?

Reply:
Since we are already collecting the master password during this step, we can go ahead and set the needed keys to avoid the extra unlock step.

Reply:
Okay - just thought we agreed on having them re-enter it anyway (similar to how when you change a password you have to re-enter anyway)

Reply:
Yea, I think we originally did say that. After implementing this part of it, I realized we could optimize that flow a bit.